README.md

```diff
@@ -56,8 +56,106 @@ If `symfony/flex` is not available, or you prefer manual setup, follow these ste
 
 ## Usage
 
+### API Usage
+
 After setting up the bundle, you can use the Storyblok client within your Symfony application to interact with the
 Storyblok CMS API.
 
 For detailed usage and examples, please refer to
 the [Storyblok API SDK documentation](https://github.com/sensiolabs-de/storyblok-api).
+
+### Webhooks
+
+Storyblok Webhooks allow your Symfony application to react to events like content changes. This bundle provides easy
+setup for handling these Webhooks.
+
+#### Configuration
+
+To enable Webhooks, add the following route to your application:
+
+```yaml
+# config/routes/storyblok.yaml
+storyblok:
+    resource: '@StoryblokBundle/config/routes.php'
+```
```

Comment:
we can do this via recipe, right?

Reply:
It will be yea but it will be commented out in the recipe.